docs/subnets/reference-elastic-subnets-parameters.md

```diff
@@ -0,0 +1,224 @@
+
+* `MaxConsumptionRate` must be larger or equal to `MinConsumptionRate`.
+* `MaxConsumptionRate` must be smaller or equal to `PercentDenominator`[^1].
+
+### `MinValidatorStake`
+
+`MinValidatorStake` has type `uint64`. It's the minimum amount of funds required to become a validator.
+The following constraints apply:
+
+* `MinValidatorStake` must be larger than zero
+* `MinValidatorStake` must be smaller or equal to `InitialSupply`
+
+### `MaxValidatorStake`
+
+`MaxValidatorStake` has type `uint64`. It's the maximum amount of funds a single
+validator can be allocated, including delegated funds.
+The following constraints apply:
+
+* `MaxValidatorStake` must be larger or equal to `MinValidatorStake`
+* `MaxValidatorStake` must be smaller or equal to `MaximumSupply`
+
+### `MinStakeDuration`
+
+`MinStakeDuration` has type `uint32` and it's the minimum number of seconds a staker can stake for.
+The following constraints apply:
+
+* `MinStakeDuration` must be larger than zero.
+
+### `MaxStakeDuration`
+
+`MaxStakeDuration` has type `uint32` and it's the maximum number of seconds a staker can stake for.
+The following constraints apply:
+
+* `MaxStakeDuration` must be larger or equal to `MinStakeDuration`.
+* `MaxStakeDuration` must be smaller or equal to `GlobalMaxStakeDuration`.
+
+`GlobalMaxStakeDuration` is defined in genesis and applies to both the Primary Network and all Subnets.
+<!-- markdownlint-disable MD013 -->
+<!-- vale off -->
+Its Mainnet value is $365 \times 24 \times time.Hour$.
+<!-- vale on -->
+<!-- markdownlint-enable MD013 -->
+### `MinDelegationFee`
+
+`MinDelegationFee` has type `uint32` and it's the minimum fee rate a delegator
+must pay to its validator for delegating. `MinDelegationFee` is a percentage; the
+actual fee is calculated multiplying the fee rate for the delegator reward.
+The following constraints apply:
+
+* `MinDelegationFee` must be smaller or equal to `PercentDenominator`.
+
+The `MinDelegationFee` rate applies to Primary Network as well. Its Mainnet value is $2\%$.
+
+### `MinDelegatorStake`
+
+`MinDelegatorStake` has type `uint64` and it's the minimum amount of funds required to become a delegator.
+The following constraints apply:
+
+* `MinDelegatorStake` must be larger than zero.
+
+### `MaxValidatorWeightFactor`
+
+`MaxValidatorWeightFactor` has type `uint8` and it's the factor which calculates
+the maximum amount of delegation a validator can receive. A value of 1
+effectively disables delegation. You can find more details about it in the
+Delegators Weight Checks section.
+The following constraints apply:
+
+* `MaxValidatorWeightFactor` must be larger than zero.
+
+### `UptimeRequirement`
+
+`UptimeRequirement` has type `uint32` and it's the minimum percentage of its
+staking time that a validator must be online and responsive for to receive a
+reward.
+The following constraints apply:
+
+* `UptimeRequirement` must be smaller or equal `PercentDenominator`.
+
+## Reward Formula
```

Comment:
@dasconnor this section here is prolly not right in a reference doc and should be moved to an explaination/discussion doc. I'd ask to keep it here until I draft such a doc.